### PR TITLE
Presentation: Frontend property formatting

### PR DESCRIPTION
**Pull request by saskliutas**
Added Schema Format lookup based on property KindOfQuantity.

Part of https://github.com/iTwin/itwinjs-core/issues/5029

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

File: presentation/common/src/presentation-common/PropertyFormatter.ts [hunk 1]
```diff
@@ -79,17 +101,119 @@ export class ContentPropertyValueFormatter extends PropertyValueFormatter {
...
+  return schema.getItem<KindOfQuantity>(propKoqName);
+}
+
+async function getKoqFormat(koq: KindOfQuantity, unitSystem: UnitSystemKey) {
+  const unitSystems = getUnitSystemGroupNames(unitSystem);
+  const presentationFormat = await getKoqPresentationFormat(koq, unitSystems);
+  if (presentationFormat)
+    return presentationFormat;
+
+  return koq.defaultPresentationFormat;
```
Comment (grigasp) [inline presentation/common/src/presentation-common/PropertyFormatter.ts:155]: should attempt to use persistence unit if it's in requested unit system and only then fall back to default presentation format

File: presentation/frontend/package.json [hunk 2]
```diff
@@ -54,6 +55,7 @@
     "@itwin/core-frontend": "workspace:*",
     "@itwin/core-i18n": "workspace:*",
     "@itwin/core-quantity": "workspace:*",
+    "@itwin/ecschema-metadata": "workspace:*",
```
Comment (grigasp) [inline presentation/frontend/package.json:58]: Do we really need these deps on `presentation-frontend`? Seems like having them on `presentation-common` should be sufficient.

File: common/changes/@itwin/presentation-common/presentation-frontend_property_formatting_2023-03-01-10-34.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-common",
+      "comment": "Added Format lookup in ECSchema for formatting properties with KindOfQuantity",
```
Comment (grigasp) [inline common/changes/@itwin/presentation-common/presentation-frontend_property_formatting_2023-03-01-10-34.json:5]: should probably mention the new peer dep too

**Review by grigasp — CHANGES_REQUESTED** [review]

File: presentation/common/src/presentation-common/PropertyFormatter.ts (hunk 1 above)
Comment (saskliutas) [reply presentation/common/src/presentation-common/PropertyFormatter.ts]: Added format for persistence unit

File: presentation/frontend/package.json (hunk 2 above)
Comment (saskliutas) [reply presentation/frontend/package.json]: removed from backend too.

File: common/changes/@itwin/presentation-common/presentation-frontend_property_formatting_2023-03-01-10-34.json (hunk 3 above)
Comment (saskliutas) [reply common/changes/@itwin/presentation-common/presentation-frontend_property_formatting_2023-03-01-10-34.json:5]: Added new change file mentioning new peer dep

**Review by grigasp — APPROVED** [review]

**Pull request merged**